test/test_ops.py

```diff
@@ -180,6 +180,11 @@ def test_inplace_gradgrad(self, device, dtype, op):
 class TestCommon(JitCommonTestCase):
     exact_dtype = True
 
+    # variant testing is only done with torch.float to avoid excessive
```

Comment:
Extend supported dtype testing to include backward testing and run on all sample inputs
also extend unsupported dtype testing to run on all sample inputs
measure total performance change